lms/templates/header/navbar-logo-header.html

```diff
@@ -15,9 +15,9 @@
 
 <h1 class="header-logo">
     <div class="navbar-brand header">
-      <a href="https://web.mit.edu/" rel="noopener noreferrer" target="_blank" class="mit-link"></a>
-      <a href="${settings.MARKETING_SITE_BASE_URL}"" class="marketing-site-link"></a>
-      <img  class="logo" src="${branding_api.get_logo_url(is_secure)}" alt="${_("{platform_name} Home Page").format(platform_name=static.get_platform_name())}"/>
+      <a href="${settings.MARKETING_SITE_BASE_URL}">
```

Comment:
How flexible is this in the sense of adding a URL other than MARKETING_SITE_BASE_URL. Ideally, If possible we should send the user to https://openlearning.mit.edu/ from the trademark logo. I looked a bit but I don't see a suitable configuration for both MFEs and Legacy themes. Can you check in detail if we can set an explicit URL just for the trademark logo?

Reply:
I've looked into it, I also didn't get any suitable configuration. I think, this is same we are using for xpro. So we can use it as it is, and can pick in the future as it is required a thorough investigation.